### ISSUE TITLE
generate function not woking

### ISSUE THREAD

**Issue by aritrakrbasu** [bostrom/text-to-image#141]
Unhandled Rejection (TypeError): textToImage.generate is not a function

**Comment by bostrom** [comment]
Can you provide some more info and a code example how you're using it? 

Works fine in my tests and this [example CodeSandbox](https://codesandbox.io/s/text-to-image-example-r8h1o).

**Comment by z88kat** [comment]
Not working here either, version 2.4.4 is working just fine, but since version 3 it is broken.

**Comment by z88kat** [comment]
here is an example

let img = await textToImage.generate("AB", {
                        debug: false,
                        maxWidth: 128,
                        fontSize: 56,
                        fontFamily: 'Arial',
                        fontWeight: 'bold',
                        lineHeight: 56,
                        customHeight: 78,
                        textAlign: 'center',
                        margin: 22,
                        bgColor: "white",
                        textColor: "purple"
                    });

**Comment by bostrom** [comment]
It's not about how the generate function is called but how the library is imported. Can either of you @aritrakrbasu or @z88kat set up a CodeSandbox where you replicate the behavior so I can see what's going on? Or share a full code example including imports. Thanks!

**Comment by z88kat** [comment]
Hopefully this link works.

https://codesandbox.io/s/wonderful-kapitsa-ozqix?file=/src/index.js

**Comment by aritrakrbasu** [comment]
> It's not about how the generate function is called but how the library is imported. Can either of you @aritrakrbasu or @z88kat set up a CodeSandbox where you replicate the behavior so I can see what's going on? Or share a full code example including imports. And what environment are you using? Browser or NodeJs? What version? Are you using vanilla JS or some transpiler or framework? Thanks!

I just solved my issue .... the fault was from my end , I used  'const textToImage = require("text-to-image");' to import the library then I  solved it by changing it to " import textToImage from 'text-to-image " . Thank you for your concern .

**Comment by bostrom** [comment]
Thanks @aritrakrbasu. I didn't realise when converting the project to TypeScript that I accidentally introduced an excess object when importing through `require('text-to-image')`. 

In version 3.x this would have worked:

```
const textToImage = require('text-to-image').default;

textToImage.generate(...);
```

but that wasn't my intention. I fixed the issue and will release 4.0.0 soon.
